frontend/app/src/store/history/consts.ts

```diff
@@ -169,12 +169,20 @@ export const useHistoryEventTypeData = createSharedComposable(() => {
         label: tc('transactions.events.history_event_subtype.return_wrapped')
       },
       {
-        identifier: HistoryEventSubType.REWARD,
```

Comment:
where did reward go?

Reply:
reward is already exist on top, it's duplicated before